src/control-plane/backend/lambda/api/common/utils.ts

```diff
@@ -520,6 +520,19 @@ function getSubnetRouteTable(routeTables: RouteTable[], subnetId: string) {
   return !isEmpty(subnetRouteTable) ? subnetRouteTable : mainRouteTable;
 }
 
+function _findVpcEndpointIndex(vpcEndpointServices: string[], serviceName: string) {
+  if (serviceName.startsWith('cn.')) {
+    const defaultS3ServiceName = serviceName.substring(3);
```

Comment:
Why name it `defaultS3ServiceName`? Is it only a specific case for S3?

I think using a generic name is better.

Reply:
done